_articles/organizations/getting-started-organizations.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: article
+title: Getting Started with Bitwarden Organizations
+categories: [organizations]
+featured: false
+popular: false
+hidden: true
+tags: [tutorial]
+---
+
+With the Bitwarden free account, and a free Organization for up to 2 users, you can experience secure sharing in no time!
+
+- [Introduction](#introduction)
+  * [What are organizations?](#what-are-organizations)
+- [Set up the First and Second User Accounts](#set-up-the-first-and-second-user-accounts)
+  * [1. Create an account for User 1](#1-create-an-account-for-user-1)
+  * [2. Verify Your Email](#2-verify-your-email)
+  * [3. Set up the Second User Account](#3-set-up-the-second-user-account)
+- [Setup and Share with Organizations](#setup-and-share-with-organizations)
+  * [4. Create a new Organization](#4-create-a-new-organization)
+  * [5. Add the Second User to your Organization](#5-add-the-second-user-to-your-organization)
+  * [6. Accepting the Invitation](#6-accepting-the-invitation)
+  * [7. Sharing within an Organization](#7-sharing-within-an-organization)
+- [Continuing with Bitwarden](#continuing-with-bitwarden)
+
+# Introduction
```

Comment:
We only use `##` and `###` withing the article itseld. `#` is already reserved for the large article title on the page.